tests/formattedcode/test_writers.py

```diff
@@ -142,7 +142,8 @@ def test_scanned_path_is_present_in_html_app_output(monkeypatch):
     assert 'Scanning done' in result.output
     html_file = open(result_file).read()
     assert '<title>ScanCode scan results for: %(test_dir)s</title>' % locals() in html_file
-    assert 'ScanCode</a> scan results for: %(test_dir)s</span>' % locals() in html_file
+    assert '<strong>scan results for:' % locals() in html_file
+    assert '<p>%(test_dir)s</p>' % locals() in html_file
```

Comment:
Checking the whole HTML file doesn't make sense. Let's insert something meaty in there. Just a question, How to include next line in the assert statement? I was having some trouble in that part.